# HG changeset patch
# User Nicolas Aguirre <aguirre.nicolas@gmail.com>
# Date 1274727596 -7200
# Node ID ae781a0decea52ba717d1b038a43b16b9edaae0c
# Parent  daf3e60729f2aee7bc837a8ad63837ccaa4fa20a
set enna theme to main window. More work will follow to fix other parts that doesn't inherit to enna->win.

diff -r daf3e60729f2 -r ae781a0decea src/bin/enna.c
--- a/src/bin/enna.c	Mon May 24 18:55:40 2010 +0200
+++ b/src/bin/enna.c	Mon May 24 20:59:56 2010 +0200
@@ -326,6 +326,7 @@
     enna->win = elm_win_add(NULL, "enna", ELM_WIN_BASIC);
     if (!enna->win)
       return 0;
+    elm_object_theme_set(enna->win, enna_config->eth);
     elm_win_title_set(enna->win, "Enna MediaCenter");
     enna->run_fullscreen = enna_config->fullscreen | run_fullscreen;
     elm_win_fullscreen_set(enna->win, enna->run_fullscreen);
